doc/cephfs/fs-volumes.rst

```diff
@@ -72,6 +72,14 @@ tries to remove MDS daemons using the enabled Ceph Manager orchestrator module.
    being used. See https://tracker.ceph.com/issues/49605#note-5 for more
    details.
 
+.. note:: If the snap-schedule Ceph Manager module is being used for a volume
+   and the volume is deleted, then the snap-schedule Ceph Manager module will
+   continue to hold references to the old pools. This will lead to the
+   snap-schedule Ceph Manager module faulting and logging errors. To remedy
+   this scenario, we recommend that the snap-schedule Ceph Manager module
```

Comment:
s/we/it is

Reply:
I had **it is** before I noticed that elsewhere it was **we**.
Anthony too didn't raise any concerns about the language.

Reply:
Zac has kinda encouraged me to follow up with my own wordsmithing PRs after merge.